f21/ps.html

```diff
@@ -1694,30 +1841,30 @@ <h2><u> Lecture / Lab 12</u></h2>
 <h2><u> Lecture / Lab 13</u></h2>
 
 
-<li > <b>Due Date: 10 December</b>  <b style="float:right">Reading: <a href="https://www.cplusplus.com/files/tutorial.pdf">Cplusplus Tutorial</a> & <a href="lab13.html">Lab 13</a></b><br>
+<li > <b>Due Date: 7 December</b>  <b style="float:right">Reading: <a href="https://www.cplusplus.com/files/tutorial.pdf">Cplusplus Tutorial</a> & <a href="lab13.html">Lab 13</a></b><br>
 
-  <p> Write a <b>C++ program</b>  that asks the user for their numerical grade and prints
+  <p> Write a <b>C++ program</b>  that asks the user for the current temperature and prints
   <ul>
-    <li>"Your letter grade is F" if it is strictly less than 60,
-    <li> "Your letter grade is C or D" if it is 60 or greater, but strictly less than 80,
-    <li> "Your letter grade is B" if it is 80 or greater, but strictly less than 90, and
-    <li> "Your letter grade is A" otherwise.
+    <li> If the given temperature is 32 degrees or lower, print "It's freezing!"
```

Comment:
Let's change this so it's not so 1-1 with the code. 
Please edit so it looks something like

"It's freezing!" for temperatures of 32 degrees or lower
"It's cold!" for temperatures less than 68 and greater ...
...

Reply:
Got it.